### PR TITLE
Snippet injection Test update

### PR DESCRIPTION
**Pull request by siyuniu-ms**

### CODE REVIEW

File: smoke-tests/apps/SnippetInjection/src/smokeTest/resources/applicationinsights.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "connectionString": "",
+  "role": {
+    "name": "testrolename",
+    "instance": "testroleinstance"
+  },
+  "sampling": {
+    "percentage": 100
+  },
+  "preview": {
+    "javaScriptSnippet":
```
Comment (heyams) [inline smoke-tests/apps/SnippetInjection/src/smokeTest/resources/applicationinsights.json:3]: should this be "browserSdkLoader", saw your doc update?

Comment (siyuniu-ms) [reply smoke-tests/apps/SnippetInjection/src/smokeTest/resources/applicationinsights.json]: haha, yes, just updated it.

**Comment by heyams** [comment]
can you update the smoke test name to reflect this renaming and be consistent?

File: agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java [hunk 2]
```diff
@@ -392,7 +392,7 @@ private static Set<Feature> initStatsbeatFeatureSet(Configuration config) {
     if (!config.preview.instrumentation.jaxrsAnnotations.enabled) {
       featureList.add(Feature.JAXRS_ANNOTATIONS_DISABLED);
     }
-    if (config.preview.javaScriptSnippet.enabled) {
+    if (config.preview.browserSdkLoader.enabled) {
       featureList.add(Feature.JAVASCRIPT_SNIPPET);
```
Comment (trask) [inline agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java:396]: let's rename the "feature" also
```suggestion
      featureList.add(Feature.BROWSER_SDK_LOADER);
```

File: agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/SnippetConfiguration.java [hunk 3]
```diff
@@ -21,7 +21,7 @@ public class SnippetConfiguration {
   // visible for testing
   static String readSnippet() {
     ClassLoader classLoader = SnippetConfiguration.class.getClassLoader();
-    String resourceName = "javascript-snippet.txt";
+    String resourceName = "browserSdkLoader-snippet.txt";
```
Comment (trask) [inline agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/SnippetConfiguration.java:24]:
```suggestion
    String resourceName = "browser-sdk-loader-snippet.txt";
```

**Review by heyams — APPROVED** [review]

**Pull request merged**